docscan/session/create/check/document_authenticity_test.go

```diff
@@ -3,17 +3,77 @@ package check
 import (
 	"encoding/json"
 	"fmt"
+	"testing"
+
+	"gotest.tools/v3/assert"
 )
 
 func ExampleRequestedDocumentAuthenticityCheckBuilder() {
-	check, err := NewRequestedDocumentAuthenticityCheckBuilder().Build()
+	docAuthCheck, err := NewRequestedDocumentAuthenticityCheckBuilder().WithManualCheckFallback().Build()
```

Comment:
Should we also have coverage without manual fallback config?

Reply:
Added in 1e55c77